modules/ROOT/pages/dataweave-formats-java.adoc

```diff
@@ -302,3 +302,142 @@ This format supports the following MIME types.
 | MIME Type
 |`*/java`
 |===
+
+
+[[java_support]]
+== Java Support
+
+Starting with version 2.6, DataWeave supports Java 17. Java 17 adds restrictions in encapsulation and reflective access that affects the xref:dataweave-formats-java.adoc[]. Because DataWeave uses Java's reflection API to read and write Java objects, ensure that the objects your application uses are Plain Old Java Objects (POJOs) containing:
+
+* Default constructor (required if your class needs to be instantiated)
+* Getters for all properties (required if your class needs to be read)
+* Setters for all properties (required if your class needs to be instantiated)
+
```

Comment:
I'm not sure that these clarifications ("required if your..") are needed.
I would like to keep it simple:
```
* Default constructor
* Getters for all properties
* Setters for all properties
```